globalconfig.yml

```diff
@@ -13,7 +13,7 @@ vm_readme_file: "${HOME}/Desktop/README.md"
 vm_hostname: "qmobile"
 vm_user: "max"
 vm_password: "moritz"
-vm_memory: 1024
+vm_memory: 2048
```

Comment:
Out of interest, was there a particular reason for this change?

Reply:
Yes: one of the codes (I can't remember which one, sorry) was taking forever to compile, and I found that it had filled both RAM and swap space. I think this change is reasonable, and can be overridden anyway if a user is careful, but if you're skeptical I can try again with 1 GB.

Reply:
I think the code was BigDFT...

Reply:
Never used it but I already hate BigDFT lol. No I don't know of any particular reason why this would be an issue

Reply:
Well to be fair, 1 GB or RAM is not a lot... :)

Reply:
You just need to keep in mind that this RAM is something that is snatched away from the rest of the operating system, i.e. on a laptop with 4GB of RAM that would be half the RAM.
Increasing RAM is always possible, but if the RAM is set too high by default, we may be interfering with the host OS.

I'm not strongly against this change, but I would agree to make it only if needed.

Reply:
* While I can't find official system requirements for Ubuntu 18.04, 1 GB RAM seems to be right at the very bottom of the scale, and not recommended.
* At boot, the machine uses about 750 MB with nothing open, rising to 1 GB on its own after 10 minutes.
* We must provide something that works and is usable, with clear requirements. It is the user's job to make sure that their PC is powerful enough.
* On the other hand, I understand if we want to cater to users with more limited resources. In that case I suspect we need to take other actions, such as [switching to Lubuntu](https://github.com/marvel-nccr/quantum-mobile/issues/84) or Xubuntu. ([Discussion here](https://github.com/marvel-nccr/quantum-mobile/issues/120).)
* I'll try to make the machine with 1.5 GB as a compromise.